src/core/lib/event_engine/windows/windows_engine.cc

```diff
@@ -556,7 +559,8 @@ bool WindowsEventEngine::CancelConnect(EventEngine::ConnectionHandle handle) {
   }
   auto* connection_state = reinterpret_cast<ConnectionState*>(handle.keys[0]);
   grpc_core::MutexLock state_lock(&connection_state->mu());
-  // The connection cannot be cancelled if the deadline timer is already firing.
+  // The connection cannot be cancelled if the deadline timer is already
```

Comment:
Are you setting your word wrap to < 80 characters? The `.` falls on column 80. This should be reverted.

Reply:
reverted